dev/jenkins/build.sh

```diff
@@ -30,3 +30,8 @@ then
   git clean -fdx
 fi
 mvn -Duser.home=/home/jenkins -T 4C clean install -Pdeveloper -Dmaven.javadoc.skip -Dsurefire.forkCount=${ALLUXIO_BUILD_FORKCOUNT} $@
+
+if [ -n "${ALLUXIO_SONAR_ARGS}" ]
```

Comment:
semicolon?
```suggestion
if [ -n "${ALLUXIO_SONAR_ARGS}" ];
```

Reply:
I just copied whatever we do in this file and the other file, and none of them use the semicolon.

Reply:
It might be if the `fi` is on the same line